docs/Configuring-Pantheon/Networking/Managing-Peers.md

```diff
@@ -0,0 +1,66 @@
+description: Managing Pantheon peers 
+<!--- END of page meta data -->
+
+# Managing Peers 
+ 
+## Limiting Peers
+
+Limiting peers reduces the bandwidth used by Pantheon. It also reduces the CPU time and disk access 
+used to manage and respond to peers.  
+ 
+Use the [`--max-peers`](../../Reference/Pantheon-CLI-Syntax.md#max-peers) command line option to reduce 
+the maximum number of peers. The default is 25.
+
+## No Discovery
+
+The [`--discovery-enabled`](../../Reference/Pantheon-CLI-Syntax.md#discovery-enabled) command line option 
+can be used to disable P2P peer discovery.
+Set this option to `false` if you are running a test node or a network with fixed nodes.
+
+Use the [`admin_addPeer`](../../Reference/JSON-RPC-API-Methods.md#admin_addpeer) JSON-RPC method to add peers
+or configure [static nodes](#static-nodes) when [`--discovery-enabled`](../../Reference/Pantheon-CLI-Syntax.md#discovery-enabled) 
+is set to false. 
+
+## Static Nodes
+
+Use a `static-nodes.json` file to configure static nodes. The node connects to the listed nodes on startup.
```

Comment:
what is a static node ? What is the goal of having some, what's the use case ? In what case is it mandatory ?

Reply:
Its a Geth thing, and is wanted by Kaleido - what is it used for 🤷‍♂️ It effectively says "on startup, attempt to connect to nodes X-Y-Z" (and continue to do so if you lose connection).
This is SLIGHTLY different to permissioning, which says "I am allowed to talk to Nodes X-Y-Z", but does not explicitly try to contact X-Y-Z (I don't think).